internal/support/scheduler/rest_intervalaction.go

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright 2019 VMware Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package scheduler
+
+import (
+	"net/http"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg"
+	"github.com/edgexfoundry/edgex-go/internal/support/scheduler/operators/intervalaction"
+)
+
+func restGetIntervalAction(w http.ResponseWriter, r *http.Request) {
+	if r.Body != nil {
+		defer r.Body.Close()
+	}
+	op := intervalaction.NewAllExecutor(dbClient, Configuration.Service.MaxResultCount)
+	intervalActions, err := op.Execute()
+	if err != nil {
```

Comment:
I think we need to add some error type checks so that we are returning the correct responses. The RAML looks like we need to account for `413` which is the `ErrLimitExceeded` error.

https://github.com/edgexfoundry/edgex-go/blob/ba8d5362ba3742b919d0ac002fe8d7221c5310a2/api/raml/support-scheduler.raml#L192-L204

Reply:
Will depend of location of config max result check

Reply:
413 is returned on line 36 below. Config MaxResult check is inside the operator.